deployment/postgres-init-db/sql/tables/actions/action_run.sql

```diff
@@ -0,0 +1,79 @@
+create table actions.action_run (
+  id integer generated always as identity,
+
+  settings jsonb not null,
+  parameters jsonb not null,
+  logs text,
+  error jsonb,
+  results jsonb,
+  status actions.action_run_status not null default 'pending',
+
+  action_definition_id integer not null,
+
+  created_by text,
+  created_at timestamptz not null default now(),
+  duration integer,
+
+  constraint action_run_synthetic_key
+    primary key (id),
+  foreign key (action_definition_id)
+    references actions.action_definition (id)
+    on delete set null,
```

Comment:
Bug: `action_definition_id` is `not null`. That makes sense, as what is the purpose of an action run without a corresponding action? This should be `on delete cascade`
```suggestion
  foreign key (action_definition_id)
    references actions.action_definition (id)
    on delete cascade,
```